circuits-circom/circuits/common-v2/regexes/body_hash_regex_v2.json

```diff
@@ -0,0 +1,20 @@
+{
+    "parts": [
+        {
+            "is_public": false,
+            "regex_def": "((\r\n)|^)(d|D)(k|K)(i|I)(m|M)-(s|S)ignature:( )?"
+        },
+        {
+            "is_public": false,
+            "regex_def": "((a|b|c|d|e|f|g|h|i|j|k|l|m|n|o|p|q|r|s|t|u|v|w|x|y|z)+=[^;]+;(\r|\n|\u0009| )+)+bh="
```

Comment:
Should this rather be `(\r\n\u0009| )?`
I.e. Either a `\r\n\u0009` or `space`. And only one of any of them?